department-of-veterans-affairs/abd-vro
# Change Contention Classification Lookup Table formatting to use corpDB column names

## What was the problem?

Need a consistent way to get the data from .csv into a usable lookup table. Agreed upon names with the team and now new versions of the LUT going forward should follow this format.

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/226

## How does this fix it?[^1]


## How to test this PR
- Run `pytest` from `domain-cc/cc-app/`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
